content/en/docs/ibm/deploy/install-kubeflow.md

```diff
@@ -73,13 +73,13 @@ get the best experience from Kubeflow.
     kubectl get storageclasses | grep block
     ```
 
-6. Set the Block Storage as the default storageclass.
+6. Set the Block Storage as the default storageclass. 
```

Comment:
Line 76 - same as in line 82: "storage class" (two words, instead of one)

`    # List all the (default) storage classes`

I also found that IBM uses two words in its docs: https://www.ibm.com/cloud/architecture/content/course/ibm-cloud-private-persistent-storage/defining-storage-classes:

> Default storage class
>
> A default storage class can dynamically provision storage when a storage class isn't specified.
```

Reply:
thanks @8bitmp3